libraries/matrix/test/src/main/kotlin/io/element/android/libraries/matrix/test/FakeMatrixClient.kt

```diff
@@ -170,8 +170,8 @@ class FakeMatrixClient(
         clearCacheLambda()
     }
 
-    override suspend fun logout(ignoreSdkError: Boolean, forced: Boolean): String? = simulateLongTask {
-        return logoutLambda(ignoreSdkError, forced)
+    override suspend fun logout(userInitiated: Boolean, ignoreSdkError: Boolean): String? = simulateLongTask {
+        return logoutLambda(ignoreSdkError, userInitiated)
```

Comment:
There is a swap in the parameters, this is not a big deal, but it can be confusing. Also when `userInitiated` is `false`, `ignoreSdkError` does not have sense.

Maybe we should create a small sealed interface to handle this:

```kotlin
    sealed interface LogoutParameter {
        data class UserInitiated(val ignoreSdkError: Boolean) : LogoutParameter
        data object ServerInitiated : LogoutParameter
    }
```

Not a blocker for this PR though, I can do it later, just sharing the idea here :)

Reply:
Can we add this in a separate PR then? So I can merge this and add it to the current release.